src/core/components/inputs/text-input/text-input.component.scss

```diff
@@ -0,0 +1,73 @@
+@use "sass:list";
+@import "src/global-styles/_theme-colors.scss";
+
+$flavors: (
+  primary: (
+    $primary-default
```

Comment:
Just use the value and don't use a list with a single element.

Reply:
But I like it for more generic naming further down, when setting the color. Removing it means more work todo when assigning the colors.